types/mokapi/faker.d.ts

```diff
@@ -0,0 +1,68 @@
+import { JSONValue } from ".";
+
+/**
+ * Creates a fake based on the given schema
+ * https://mokapi.io/docs/javascript-api/mokapi-faker/fake
+ * @param schema schema - OpenAPI Schema object contains definition of a type
+ * @example
+ * export default function() {
+ *   console.log(fake({type: 'string'}))
+ *   console.log(fake({type: 'number'}))
+ *   console.log(fake({type: 'string', format: 'date-time'}))
+ *   console.log(fake({type: 'string', pattern: '^\d{3}-\d{2}-\d{4}$'})) // 123-45-6789
+ * }
+ */
+export function fake(schema: Schema): JSONValue;
```

Comment:
There is no package `mokapi` on npm, but this is a module. How is this package intended to be used? What are these types for?

Reply:
Mokapi is a Golang app with an embedded JavaScript engine. The types are intended to provide a better developer experience when writing JavaScript code for the app.

Reply:
Can you provide any documentation about this?